va_explorer/templates/users/user_detail.html

```diff
@@ -42,7 +42,25 @@
         {%  endfor %}
     </div>
 </div>
-<br/>
+{% comment "Note that Django does not support filtering of query sets in templates" %}
+{% endcomment %}
+{% for group in user.groups.all %}
+  {% if group.name == "Field Workers" %}
```

Comment:
Wondering if we should have a generic `is_field_worker()` method (or `@property`) on the user so we don't need to loop and check for the string everywhere?